### PR TITLE
CMake support

### PR DESCRIPTION
**Pull request by dabrahams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,43 +5,111 @@ Hylo is a programming language that leverages [mutable value semantics](https://
...
+
+2.  **Build**: 
+
+    ```
+    cmake --build <build-directory>
+    ```
+
+3. **Test** (requires `-DBUILD_TESTING=1` in step 1):
 
-### Prerequisites
```
Comment (peter-evans) [inline README.md:18]: There's a reference to this section on L147.

That paragraph should probably be updated to something like this:
```diff
That integrated terminal is connected to the Devcontainer, as if by ssh.
- Use the `make-pkgconfig` tool to configure LLVM's library description (see steps 3 in [prerequisites](#prerequisites)).
+ Follow the steps to [build with Swift Package Manager](#building-with-swift-package-manager).
You can now run `swift test -c release` to build and test for Linux.
```

**Review by peter-evans — APPROVED** [review]
LGTM

I built the devcontainer in a Codespace and checked that I could build and test with the swift package manager. All ok 👍

File: README.md (hunk 1 above)
Comment (dabrahams) [reply README.md:18]: @peter-evans How's [this](https://github.com/hylo-lang/hylo/pull/1439/commits/2eaeb06662aecfad1a29429c1468cdf1de5b00ac)?

**Comment by kyouko-taiga** [comment]
🎉 🥳

**Pull request merged**